commands/sdcli_version

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env bash
 
 #Lists the versions of the various languages and applications installed on SDCLI's image
-
+# suppress color coding if TERM was not set
+: ${TERM:=dumb}
```

Comment:
Question: Pardon my ignorance, but what does the leading `:` do?

Reply:
It's a builtin that is equivalent to `true`. 
[gory details](http://tldp.org/LDP/abs/html/special-chars.html) in "Provide a placeholder where a binary operation is expected"